engine/admin/logging/awslogs.md

```diff
@@ -77,6 +77,115 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### awslogs-multiline-pattern
+
```

Comment:
It still needs it even if so, because 17.06 is not released yet. The code is:

```
{% include edge_only.md section="option" %}
```

Or maybe "feature" instead of "option" if that fits better.